dashboard/lib/projects_list.rb

```diff
@@ -109,8 +112,18 @@ def get_published_project_data(project)
         # Note that we are using the new :project_type field rather than extracting
         # it from :value. :project_type might not be present in unpublished projects.
         type: project[:project_type],
-        publishedAt: project[:published_at]
+        publishedAt: project[:published_at],
+        studentName: student_name(project),
+        studentAge: student_age(project),
       }.with_indifferent_access
     end
+
+    def student_age(project)
+      ((Date.today - project[:birthday]) / 365).to_i
+    end
+
+    def student_name(project)
+      project[:name] ? project[:name][0] : ''
```

Comment:
You could follow [this pattern](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/user.rb#L381-L388), defining a class method that accepts a string (the student name) and an instance method that calls the class method (passing `name`).

That said, no worries.